build/testdata/bundles/wordpress/porter.yaml

```diff
@@ -33,23 +34,37 @@ parameters:
 install:
   - helm:
       description: "Install Wordpress"
-      name: "{{bundle.parameters.wordpress-name }}"
+      name: "{{ bundle.parameters.wordpress-name }}"
       chart: stable/wordpress
       namespace: "{{ bundle.parameters.namespace }}"
       replace: true
       set:
         wordpressPassword: "{{ bundle.parameters.wordpress-password }}"
-        externalDatabase.database: "{{ bundle.dependencies.mysql.parameters.database-name }}"
+        #externalDatabase.database: "{{ bundle.dependencies.mysql.parameters.database-name }}"
```

Comment:
These are just commented out temporarily until #432 puts it back into place. PR was getting huge.